multiqc/modules/humid/humid.py

```diff
@@ -117,3 +117,5 @@ def process_stats(stats, s_name):
     except AssertionError:
         log.warning(f"HUMID stats looked wrong, skipping: {s_name}")
         return False
+    finally:
+        return True
```

Comment:
Do you not need to return `stats`? Otherwise this function isn't doing anything..

```suggestion
    return stats
```

Maybe would be better still to include these functions inside the `MultiqcModule` (up one level of indentation) and handle everything there.

Reply:
Oof, just noticed that you have two different functions both called `parse_stat_file` within one file - one inside the object scope and one outside. This needs refactoring... Sorry, must have missed this in the initial review.